app_2017/templates/Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'newrelic_rpm'
 gem 'sentry-raven'
 
 # Frontend Common
-gem 'webpacker'
+gem 'webpacker', '~> 3.2.0'
```

Comment:
does this break other stuff? no worries if so, just wondering

Reply:
Since starting this, webpacker upgraded several versions, I'm just hoping it brings us consistency (or avoids a non functional script).  Perhaps, I don't have a firm grasp on it, but for some reason the config on here worked alright for 3.2 but not 3.1, although most of this dev was I believe on 3.0.  Weird stuff.